lib/dpul_collections/utilities.ex

```diff
@@ -0,0 +1,36 @@
+defmodule DpulCollections.Utilities do
+  alias DpulCollections.IndexingPipeline
+  alias DpulCollections.IndexingPipeline.Figgy
+
+  def reindex_solr(cache_version) do
```

Comment:
Should these functions be part of the Consumer namespace? So like, `Figgy.IndexingConsumer.Restart()` or `Reset()` or `ResetTo(nil)` or something?